src/Symfony/Component/Debug/Tests/FatalErrorHandler/ClassNotFoundFatalErrorHandlerTest.php

```diff
@@ -61,7 +61,7 @@ public function testHandleClassNotFound($error, $translatedMessage, $autoloader
         }
 
         $this->assertInstanceOf('Symfony\Component\Debug\Exception\ClassNotFoundException', $exception);
-        $this->assertSame($translatedMessage, $exception->getMessage());
+        $this->assertRegExp($translatedMessage, $exception->getMessage());
```

Comment:
why switching this to a regex, while all your regexps are anchored on both sides and contain only static parts ?

Reply:
He added a `.*` in lines 103 and 121.

Reply:
error messages differ slightly for these two cases when the ErrorCatcher component is present: https://travis-ci.org/symfony/symfony/jobs/551399461#L4725